rollouts/manifests/crds/containercluster.yaml

```diff
@@ -0,0 +1,1351 @@
+# Copyright 2023 Google LLC
```

Comment:
I see the script generates `controller/controller.yaml` and `crds/crds.yaml`, but where does this manifest come from?

Reply:
They are originally coming from kubebuilder generated config under `config` directory. Think of that as the `DRY` or source config and this being a WET package that will be distributed to the end-users.

Reply:
I suppose the part I'm confused by is the other manifests in this directory appear to be regenerated from their dry config every time (by `create-manifests.sh`), whereas this one appears to be a static wet config. Is this because there is no "source of truth" dry config for this CRD?

Reply:
Oh I am sorry, I mis-read your question. 

> Is this because there is no "source of truth" dry config for this CRD?

You are right, there is no SOT dry config for this CRD. This CRD exists on clusters that have KCC installed (for example config-controller clusters). So, I added this for now to complete the package because the package can be installed on any GKE clusters. Eventually, I want to find an elegant solution where this should be an optional installed but has bunch of dependences such as RBAC and also making controller code aware of the presence/absence of the CRDs, but we plan to address further down the line.

Reply:
Yeah, this is probably ok for now, but we should find a better solution here. This will just install the CRD without the controller. It can also create problems if users try to install this package on a cluster that already have Config Connector installed.